docker-compose.yml

```diff
@@ -19,5 +19,8 @@ services:
       HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
       HASURA_GRAPHQL_ADMIN_SECRET: key
       HASURA_GRAPHQL_JWT_SECRET: '${SKILLZ_GRAPHQL_JWT_SECRET?SKILLZ_GRAPHQL_JWT_SECRET}'
+      #Uncomment this line if youre using Linux
+      #HASURA_ACHIEVEMENTS_ENDPOINT: http://172.17.0.1:3000/api/achievement
+      HASURA_ACHIEVEMENTS_ENDPOINT: http://host.docker.internal:3000/api/achievement
```

Comment:
We should use 
```
HASURA_ACHIEVEMENTS_ENDPOINT: '${SKILLZ_HASURA_ACHIEVEMENTS_ENDPOINT?SKILLZ_HASURA_ACHIEVEMENTS_ENDPOINT}'
``` 
and add a section to the readme which explains wether to use the IP of `host.docker.internal` depending on the Operting System